### PR TITLE
[Config][DX] improve file loader error message for missing resource

### PR DESCRIPTION
**Pull request by cordoval**
| Q | A |
| --- | --- |
| Bug Fix? | y |
| New Feature? | n |
| BC Breaks? | n |
| Deprecations? | n |
| Tests Pass? | y |
| Fixed Tickets | #11958 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/Config/Exception/FileLoaderLoadException.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function __construct($resource, $sourceResource = null, $code = null, $pr
         if ('@' === $resource[0]) {
             $parts = explode(DIRECTORY_SEPARATOR, $resource);
             $bundle = substr($parts[0], 1);
-            $message .= ' '.sprintf('Make sure the "%s" bundle is correctly registered and loaded in the application kernel class.', $bundle);
+            $message .= ' '.sprintf('Make sure the "%s" bundle is correctly registered and loaded in the application kernel class. If the bundle is registered, make sure the folder "%s" is not empty.', $bundle, $parts[1]);
```
Comment (stof) [inline src/Symfony/Component/Config/Exception/FileLoaderLoadException.php]: what happens if ``$parts[1]`` is not defined ?

Comment (cordoval) [inline src/Symfony/Component/Config/Exception/FileLoaderLoadException.php]: i will adapt accordingly @stof after that it should be good to go right?

Comment (stof) [inline src/Symfony/Component/Config/Exception/FileLoaderLoadException.php]: IMO, yes

**Comment by cordoval** [comment]
@stof :+1: LGTM

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @cordoval.

**Comment by cordoval** [comment]
would be nice to see more easy picks :+1: :older_man:

**Pull request merged**